java/src/main/kotlin/io/spine/validation/java/generate/OptionApplicationCode.kt

```diff
@@ -37,7 +37,7 @@ import io.spine.protodata.java.MethodDeclaration
  * @property fields Additional class-level fields required by the validation logic.
  * @property methods Additional class-level methods required by the validation logic.
  */
-internal class FieldOptionCode(
+internal class OptionApplicationCode(
```

Comment:
Maybe, `OptionHandlingCode`? As per the documentation.

It would be good to get rid of this "Application" part, it pulls the mind away a bit.